internal/services/appservice/helpers/linux_web_app_schema.go

```diff
@@ -968,6 +968,16 @@ func expandAutoHealSettingsLinux(autoHealSettings []AutoHealSettingLinux) *web.A
 
 	autoHeal := autoHealSettings[0]
 
+	if len(autoHeal.Actions) > 0 {
+		action := autoHeal.Actions[0]
+		result.Actions.ActionType = web.AutoHealActionType(action.ActionType)
+		result.Actions.MinProcessExecutionTime = pointer.To(action.MinimumProcessTime)
+	}
+
+	if len(autoHeal.Triggers) == 0 {
+		return result
+	}
+
 	triggers := autoHeal.Triggers[0]
 	if len(triggers.Requests) == 1 {
```

Comment:
actually instead of returning early here can we wrap the code instead?
```suggestion
	if len(autoHeal.Triggers) == 0 {

		triggers := autoHeal.Triggers[0]
		if len(triggers.Requests) == 1 {
```

Reply:
Hi @katbyte, I'm not sure that I got your point, since line 977 `len(autoHeal.Triggers) == 0`, line 979 `autoHeal.Triggers[0]` would trigger an index out of range panic right? Do you mean `if len(autoHeal.Triggers) == 1 {` for line 977?

I personal prefer return early style to reduce nested blocks' depth and avoid [arrow code](https://blog.codinghorror.com/flattening-arrow-code/), but this is HashiCorp's project so I would be happy to follow your guidance.